src/Components/Notifications/NotificationsList.tsx

```diff
@@ -53,9 +53,7 @@ export const NotificationsList: React.FC<NotificationsListProps> = ({
 
   const { state } = useNotificationsContext()
 
-  const xs = __internal__useMatchMedia(THEME.mediaQueries.xs)
-  const sm = __internal__useMatchMedia(THEME.mediaQueries.sm)
-  const isMobile = xs || sm
+  const isMobile = getENV("IS_MOBILE")
 
```

Comment:
based on @damassi [comment here](https://github.com/artsy/force/pull/13531#discussion_r1504536546) 
@nickskalkin could you, please, confirm this change is correct and we don't have isMobile for internal use only?